<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ol√°,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ver_acervo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VER ACERVO</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cadastrar_novo_livro">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CADASTAR NOVO LIVRO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sair">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>label</zorder>
   <zorder>ver_acervo</zorder>
   <zorder>cadastrar_novo_livro</zorder>
   <zorder>sair</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
